src/umockdev.vala

```diff
@@ -655,6 +655,9 @@ public class Testbed: GLib.Object {
             return;
         }
 
+        // Create a uevent for the removal of this device.
+        uevent(syspath, "remove");
```

Comment:
I think this is a good idea, however, when I try to move the uevent call to be _after_ the removal of the directories (even if the reading of the uevent file properties is split out into a call before the directories are removed), I get this error:
```
ERROR: uevent_sender_send: No such device /sys/devices/pci0000:00/0000:00:14.0/usb1/1-3
```

Which I believe is coming from this line of code: `this.ev_sender.send(devpath, action, properties);`

Reply:
Ah, you are right. That's a bit difficult to avoid, it'd need to make a copy of the uevent file (properties) first. But it doesn't hurt too badly indeed, let's go with this for now.